Source/IGListStackedSectionController.m

```diff
@@ -135,7 +135,12 @@ - (NSInteger)numberOfItems {
 }
 
 - (CGSize)sizeForItemAtIndex:(NSInteger)index {
-    IGListSectionController *sectionController = [self sectionControllerForObjectIndex:index];
+    // during interactive reordering, its possible for an item to be moved into a section beyond the last section
+    // in that case, just return the size of the current last item in the section
+    const NSInteger maxIndex = [self numberOfItems]-1;
```

Comment:
h/t to @amonshiz for catching this bug. `numberOfItems` can absolutely return `0`. I'm updating locally before landing.

Reply:
😊

Reply:
lol oops 😳